interface/backend/static/tests.py

```diff
@@ -4,7 +4,7 @@
 
 class SmokeTest(TestCase):
     def test_landing(self):
-        url = reverse('static:home')
+        url = reverse('static:open_image')
```

Comment:
I think elsewhere we are using hyphens in url ``name`` attributes

Reply:
Thanks for the remark. I replaced all underscores in url names by hyphens.